R/data.table.R

```diff
@@ -403,6 +403,7 @@ replace_order = function(isub, verbose, env) {
         i = !i
       }
     }
+    browser()
```

Comment:
we probably don't need this

Reply:
oh that's vestigial, I must have missed it, yes please edit it out (I'm off for now)